core/dovecot/Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM alpine:3.8
+FROM alpine:3.10
 # python3 shared with most images
 RUN apk add --no-cache \
     python3 py3-pip git bash \
   && pip3 install --upgrade pip
-# Shared layer between rspamd, postfix, dovecot, unbound and nginx
-RUN pip3 install git+https://github.com/usrpro/MailuStart.git#egg=mailustart
+# Shared layer between admin, rspamd, postfix, dovecot, unbound and nginx
+RUN pip3 install git+https://github.com/Mailu/MailuStart.git#egg=mailustart
+# Shared layer between dovecot and postfix
+RUN pip3 install git+https://github.com/Nebukadneza/Podop.git@fix_py37
```

Comment:
I will merge your PR so we don't have to pip install from Github, which is terrible.